### PR TITLE
Add support for Integer to string tag enumerations

### PR DESCRIPTION
**Pull request by oliver-walker-hms**

### CODE REVIEW

**Comment by alexjhawk** [comment]
You should add a commit to increment the package.html (and any other) version number(s), then a RELEASE commit with updated changelog. Similar to https://github.com/hms-networks/sc-flexy-datapoint-lib/pull/16/commits

File: src/com/hms_networks/americas/sc/historicaldata/HistoricalDataManager.java [hunk 1]
```diff
@@ -184,8 +186,9 @@ private static boolean convertStrToBool(String strBool) {
    * @param line line to parse
    * @return data point
    * @throws IOException if unable to access tag information
+   * @throws JSONException
```
Comment (alexjhawk) [inline src/com/hms_networks/americas/sc/historicaldata/HistoricalDataManager.java]: need description/reason for exception

File: src/com/hms_networks/americas/sc/historicaldata/HistoricalDataManager.java [hunk 2]
```diff
@@ -125,8 +126,9 @@ public static void exportHistoricalToFile(
    * @param filename historical file to parse
    * @return data points parsed
    * @throws IOException if unable to access or read file
+   * @throws JSONException
```
Comment (alexjhawk) [inline src/com/hms_networks/americas/sc/historicaldata/HistoricalDataManager.java]: need description/reason for exception

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,29 +1,42 @@
...
-### Major Changes
-- Feature: Add support for string tags
-- Bug Fix: Fix improper time comparison
-
-### Minor Changes
-- Feature: Make current time with offset method public
-
-## v1.0
-Initial release
+# Ewon Flexy Tag Data Library Changelog
```
Comment (alexjhawk) [inline CHANGELOG.md:1]: This commit name might be better as "Updated changelog for previous updates"

**Review by alexjhawk — CHANGES_REQUESTED** [review]

**Review by alexjhawk — APPROVED** [review]

**Review by it-hms — APPROVED** [review]

**Pull request merged**